.env.example

```diff
@@ -35,6 +35,7 @@ TWILIO_APPLICATION_SID=
 TWILIO_AUTH_TOKEN=
 TWILIO_MESSAGE_SERVICE_SID=
 TWILIO_STATUS_CALLBACK_URL=
+TWILIO_SQS_QUEUE_URL=
```

Comment:
I don't think we should make these optional items in `.env.example` (feel free to prefer something different) but we should definitely add them to `docs/REFERENCE-environment_variables.md`. In this case, probably include a link the SQS twiliio blog post https://www.twilio.com/blog/2017/07/handling-high-volume-inbound-sms-and-webhooks-with-twilio-functions-and-amazon-sqs.html